### DISCUSSION TITLE
What musical instrument has 47 strings and seven pedals?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#325] [Galaxy Trivia] [answered]
**Hint:** Guinness

**Comment by trevorwang** [comment, marked as answer]
Just asked ChatGPT and it told me that's a harp

**Reply by ParanoidUser** [reply]
Bingo! 🍀🍀One more to go for the **Galaxy Brain** badge. 😊
![image](https://user-images.githubusercontent.com/5120290/225925216-5500f5e2-0eb8-4cdc-a238-4626eba6f612.png)

**Comment by benjaminjacob7** [comment]
It's the modern pedal harp with its 47 strings and those fascinating 7 pedals that raise or lower the strings. The sound it produces is simply enchanting! And speaking of unique instruments, I'm head over heels for the [concertina instrument](https://mcneelamusic.com/concertina-for-sale)!  It's become my go-to pastime, and I'm constantly honing my skills. Such a charming instrument!
